src/OpenTelemetry/Metrics/MeterProviderSdk.cs

```diff
@@ -45,6 +46,11 @@ internal sealed class MeterProviderSdk : MeterProvider
             int maxMetricPointsPerMetricStream,
             IEnumerable<MetricReader> readers)
         {
+            OpenTelemetrySdkEventSource.Log.MeterProviderSdkEvent("Setup started.");
```

Comment:
such messages are not necessarily useful. Lets add very specific messages in the logs. MeterProvider initialized successfully/failed with ex(exception message) etc.

Reply:
I was thinking since this verbose logging, we can have some logs to mark the beginning of `ctor` as well instead of otherwise just inferring that from the absence of logs.